### PR TITLE
Setup tutor list item views

### LINKED ISSUE #94: Setup list item details view for the Tutor Home listview
Take a look at what Connor did in this PR: https://github.com/ericdesj/KRUSH/pull/91

A list item should be clickable and bring the user to a new fragment. The data should also be passed to the child fragment using Bundle.

### PR DESCRIPTION
**Pull request by Mons1Oerjan**
### Designed and implemented the following views:
- Tutor Upcoming Sessions list item view
- Tutor Session History list item view

This closes #94 

krush-1

### Note
The video displays all sessions in the `upcoming` sessions list view in the video for testing / display purposes only. I added the `future events only` part back to the query after making the video.

### CODE REVIEW

**Comment by Mons1Oerjan** [comment]
### Added cancel button
krush-2

### Added delete functionality
krush-3

File: app/src/main/res/layout/tutor_home_upc_session_details.xml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        android:layout_marginTop="16dp"
+        android:text="school" />
+
+    <Button
+        android:id="@+id/cancel_session_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/tutor_history_details_school"
+        android:layout_centerHorizontal="true"
+        android:layout_marginTop="64dp"
```
Comment (ericdesj) [inline app/src/main/res/layout/tutor_home_upc_session_details.xml]: Add the following to make the button consistent with the rest of the app:

```
        android:paddingRight="25dp"
        android:paddingLeft="25dp"
        android:minWidth="170dp"
```

File: app/src/main/java/cs/dal/krush/tutorFragments/TutorSessionsDetailsFragment.java [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        schoolField.setText("School: " + school);
+        if (imgPath != null && !imgPath.isEmpty()) {
+            Bitmap profilePic = BitmapFactory.decodeFile(imgPath);
+            studentPicture.setImageBitmap(profilePic);
+        }
+
+
+        //Build the "Are you sure?" dialog window:
+        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
+        builder.setMessage("Are you sure?").setTitle("Cancel Session");
```
Comment (ericdesj) [inline app/src/main/java/cs/dal/krush/tutorFragments/TutorSessionsDetailsFragment.java]: I would specify in the text (even if its in the title) "Are you sure want to cancel this session?"

File: app/src/main/res/layout/tutor_home_upc_session_details.xml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        android:background="@drawable/button_cancel"
+        android:text="Cancel Session" />
```
Comment (ericdesj) [inline app/src/main/res/layout/tutor_home_upc_session_details.xml]: I would just make this 'Cancel' you specify the action in the dialog box.

File: app/src/main/res/drawable/button_cancel.xml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle" android:padding="10dp">
+    <solid android:color="@color/colorNegativeAction"/>
```
Comment (ericdesj) [inline app/src/main/res/drawable/button_cancel.xml]: We're using flat UI colours for our colour palette, we should use this red `#e74c3c`

File: app/src/main/res/values/colors.xml [hunk 5]
```diff
@@ -4,4 +4,5 @@
     <color name="colorPrimaryDark">#27ae60</color>
     <color name="colorAccent">#FF4081</color>
     <color name="bottom_bar_color">#ecf0f1</color>
+    <color name="colorNegativeAction">#c7050f</color>
```
Comment (ericdesj) [inline app/src/main/res/values/colors.xml]: `#e74c3c`

File: app/src/main/res/layout/tutor_home_upc_session_details.xml (hunk 1 above)
Comment (ericdesj) [inline app/src/main/res/layout/tutor_home_upc_session_details.xml]: Set the text colour to white as well - black on red looks weird

**Comment by Mons1Oerjan** [comment]
Awesome feedback. I fixed the design issues. Here's what it looks like now:

screen shot 2017-03-25 at 1 29 37 pm
screen shot 2017-03-25 at 1 29 48 pm

**Comment by ericdesj** [comment]
That's a sexy looking cancel button - merged

**Pull request merged**